packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -396,6 +434,8 @@ class DDSMastheadComposite extends LitElement {
     target: NAV_ITEMS_RENDER_TARGET;
     hasL1: boolean;
   }) {
+    // const currentUrlPath= window.location.href;
+    const currentUrlPath = 'https://www.ibm.com/cloud/ai';
```

Comment:
The commented line should be used here, no?

Reply:
Yessir. I will uncomment that line when I'm ready to pull the PR out of draft status. Thank you for the review.